dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MethodMetric.java

```diff
@@ -38,10 +41,23 @@ public class MethodMetric extends ServiceKeyMetric {
 
     public MethodMetric(ApplicationModel applicationModel, Invocation invocation) {
         super(applicationModel, MetricsSupport.getInterfaceName(invocation));
-        this.methodName = RpcUtils.getMethodName(invocation);
         this.side = MetricsSupport.getSide(invocation);
         this.group = MetricsSupport.getGroup(invocation);
         this.version = MetricsSupport.getVersion(invocation);
+        if (isServiceLevel(applicationModel)) {
+            this.methodName = "";
+            return;
+        }
+        this.methodName = RpcUtils.getMethodName(invocation);
```

Comment:
Why does methodName need to be assigned an empty string， Will this be more concise?
if (!isServiceLevel(applicationModel)) {
        this.methodName = RpcUtils.getMethodName(invocation);      
 }